polyply/tests/test_meta_molecule.py

```diff
@@ -243,3 +243,40 @@ def test_unkown_fromat_error():
         MetaMolecule.from_sequence_file(force_field=ff,
                                         file_path=test_path,
                                         mol_name="test")
+
+def test_copy():
+    """
+    Test if a MetaMol copy has all attributes etc.
+    """
+    file_name = TEST_DATA + "/itp/PEO.itp"
+    edges = [(0,1), (1,2)]
+    nodes = [0, 1, 2]
+    attrs = {0: 'PEO', 1: 'PEO', 2: 'PEO'}
+
+    ff = vermouth.forcefield.ForceField(name='test_ff')
+    name = "PEO"
+    meta_mol = MetaMolecule.from_itp(ff, file_name, name)
+
+    copy_mol = meta_mol.copy()
+    assert list(meta_mol.nodes) == list(copy_mol.nodes)
+    assert meta_mol.force_field == copy_mol.force_field
+    assert meta_mol.molecule == copy_mol.molecule
+    for node in meta_mol.nodes:
+        for key, attr in meta_mol.nodes[node].items():
+            assert attr == copy_mol.nodes[node][key]
```

Comment:
This is not thorough enough: modify copy_mol (or mol) and assert the other is not affected